crates/sargon/src/signing/signables/signable_subintent.rs

```diff
@@ -33,7 +33,9 @@ impl Signable for Subintent {
             );
         }
 
-        let manifest = TransactionManifestV2::sargon_built(builder, network_id);
+        builder = builder.yield_to_parent(());
```

Comment:
SubintentManifest should always have at least `yield_to_parent` instruction to be valid.